### ISSUE TITLE
How to get rid of extraneous properties or set additionalProperties to FALSE ?

### ISSUE THREAD

**Issue by knowBalpreet** [tiangolo/fastapi#638]
I have the following model

```
class AccountModel(BaseModel):
  name: str
```
During validation, if the name is not there it raises an error, but in data, if I am sending 
```
{"name": "Balpreee", "age": 23 }
```
I want just name in data, and not age. 

Please help, thanks in advance!

**Comment by knowBalpreet** [comment]
It handles it implicitly, i was wrong to do `await request.json()`

**Comment by prostomarkeloff** [comment]
Age variable don't misses to the AccountModel after validation.

**Comment by tiangolo** [comment]
Yep, it should work just like that. I think you found it, right?
